src/actions/tree.ts

```diff
@@ -200,48 +223,64 @@ export const changeDateFilter = ({newMin = false, newMax = false, quickdraw = fa
 
 /**
  * NB all params are optional - supplying none resets the tip radii to defaults
- * @param  {string|number} selectedLegendItem value of the attr. if scale is continuous a bound will be used.
- * @param  {int} tipSelectedIdx the strain to highlight (always tree 1)
- * @param  {array} geoFilter a filter to apply to the strains. Empty array or array of len 2. [0]: geoResolution, [1]: value to filter to
- * @return {null} side-effects: a single action
+ * side-effects: a single action
  */
 export const updateTipRadii = (
-  {tipSelectedIdx = false, selectedLegendItem = false, geoFilter = [], searchNodes = false} = {}
-) => {
-  return (dispatch, getState) => {
+  {
+    tipSelectedIdx = false,
+    selectedLegendItem = false,
+    geoFilter = [],
+  }: {
+    /** the strain to highlight (always tree 1) */
+    tipSelectedIdx?: number | false,
+
+    /** value of the attr. if scale is continuous a bound will be used. */
+    selectedLegendItem?: string | number | false,
+
+    /** a filter to apply to the strains. Empty array or array of len 2. [0]: geoResolution, [1]: value to filter to */
+    geoFilter?: [string, string] | [],
+  } = {}
+):       (dispatch: AppDispatch, getState: () => RootState) => void => {
+  return (dispatch: AppDispatch, getState: () => RootState) => {
     const { controls, tree, treeToo } = getState();
     const colorScale = controls.colorScale;
-    const d = {
+    const d: AnyAction = {
       type: types.UPDATE_TIP_RADII, version: tree.tipRadiiVersion + 1
     };
     const tt = controls.showTreeToo;
     if (tipSelectedIdx) {
       d.data = calcTipRadii({tipSelectedIdx, colorScale, tree});
       if (tt) {
         const idx = strainNameToIdx(treeToo.nodes, tree.nodes[tipSelectedIdx].name);
-        d.dataToo = calcTipRadii({idx, colorScale, tree: treeToo});
+        d.dataToo = calcTipRadii({tipSelectedIdx: idx, colorScale, tree: treeToo});
```

Comment:
[note for posterity]

It's great we fixed this bug but note that we never call `updateTipRadii` with a `tipSelectedIdx` parameter, so this entire `if (tipSelectedIdx)` block is dead code.